qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -546,9 +574,12 @@ public void test084SymlinkToFileWithInvalidPermissionsIsRejected() throws Except
      */
     public void test085EnvironmentVariablesAreRespectedUnderDockerExec() throws Exception {
         installation = runContainer(distribution(), builder().envVars(Map.of("ELASTIC_PASSWORD", "hunter2")));
-
+        final Path autoConfigurationDir = findInContainer(installation.config, "d", "\"tls_auto_config_initial_node_*\"");
+        assert autoConfigurationDir != null : "Unable to find the auto-configured HTTP CA cert in the container";
+        hostHttpCaCert = tempDir.resolve("http_ca_3.crt");
+        copyFromContainer(autoConfigurationDir.resolve("http_ca.crt"), hostHttpCaCert);
```

Comment:
nit: The copying of the HTTP CA cert from the installation in order to make the call is a recurring pattern. Maybe move it inside the `waitForElasticsearch` function.

Reply:
I originally added this only in DockerTests#setupTest and it seemed fine, but then as you pointed out I need to do the dance everytime a test uses `runContainer` .I'll see about adding the file resolution to ServerUtils

Reply:
I did the relevant change in ServerUtils